DataStructures.Tests/MinMaxHeapTests.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DataStructures;
+using NUnit.Framework;
+
+namespace DataStructures.Tests
+{
+    [TestFixture()]
+    public static class MinMaxHeapTests
+    {
+        private static readonly object[] collectionsSource = new object[] {
+            new int[] { 5, 10, -2, 0, 3, 13, 5, -8, 41, -5, -7, -60, -12 },
+            new char[] {'e', '4', 'x', 'D', '!', '$', '-', '_', '2', ')', 'Z', 'q'},
+            new string[] { "abc", "abc", "xyz", "bcd", "klm", "opq", "ijk" }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=5421646)